tap_hubspot/hubspot.py

```diff
@@ -402,6 +402,7 @@ def store_ids_submissions(self, record: Dict):
             for form_time in forms_times:
                 guid = form_time.split(":", 1)[0]
                 self.event_state["hs_calculated_form_submissions_guids"][guid] = None
+                self.event_state["hs_calculated_form_submissions_guids"].sync()
```

Comment:
do we want to do it after the loop or at certain intervals instead of every time?